README.md

```
@@ -2,7 +2,7 @@
 
 <!-- [![Test](https://github.com/cpp-linter/clang-tools-static-binaries/actions/workflows/test.yml/badge.svg)](https://github.com/cpp-linter/clang-tools-static-binaries/actions/workflows/test.yml) -->
 [![Build](https://github.com/cpp-linter/clang-tools-static-binaries/actions/workflows/build.yml/badge.svg)](https://github.com/cpp-linter/clang-tools-static-binaries/actions/workflows/build.yml)
-![](https://img.shields.io/badge/platform-linux--64%20%7C%20win--64%20%7C%20osx--arm64%20-blue)
+![](https://img.shields.io/badge/platform-linux--64%20%7C%20win--64%20%7C%20osx--arm64%20%7C%20osx--64-blue)
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)